Cargo.toml

```diff
@@ -37,6 +37,7 @@ compiler_builtins = { version = '0.1.0', optional = true }
 # examples from normal runs.
 examples = []
 rustc-dep-of-std = ['core', 'compiler_builtins/rustc-dep-of-std']
+vendor = []
```

Comment:
Same as with the other commit: Can you explain the rationale behind it?

Also note that this breaks API, so we would have to bump the major version again. I would thus try to delay this, if we happen to agree on merging it.